roles/setup_gitea/tasks/install.yml

```diff
@@ -0,0 +1,166 @@
+---
+- name: Create namespace for Gitea
+  community.kubernetes.k8s:
+    definition: "{{ lookup('template', 'namespace.j2') | from_yaml}}"
+
+- name: Create Service Account
+  community.kubernetes.k8s:
+    definition: "{{ lookup('template', 'sa_anyuid.j2') | from_yaml}}"
+
+- name: Create Role
+  community.kubernetes.k8s:
+    definition: "{{ lookup('template', 'role_scc_anyuid.j2') | from_yaml}}"
+
+- name: Create RoleBinding
+  community.kubernetes.k8s:
+    definition: "{{ lookup('template', 'role_binding_sa_to_scc_anyuid.j2') | from_yaml}}"
+
+- name: Generate JWT secret
+  containers.podman.podman_container:
+    name: gitea_jwt
+    image: "{{ sg_gitea_image }}"
+    command: /usr/local/bin/gitea generate secret JWT_SECRET
```

Comment:
Should we have to make sure that `gitea` binary is present in the server before trying to run this command?

Reply:
I don't think so. We're running the binary from a dedicated container based on the gitea image.

Reply:
oh, that's true, sorry, closing this